packages/keystrokes/src/key-combo-state.ts

```diff
@@ -164,11 +164,6 @@ export class KeyComboState<OriginalEvent, KeyEventProps, KeyComboEventProps> {
   updateState(activeKeys: KeyPress<OriginalEvent, KeyEventProps>[]) {
     const sequence = this._parsedKeyCombo[this._sequenceIndex]
 
-    // Do nothing if no keys are pressed
-    if (activeKeys.length === 0) {
-      return
-    }
```

Comment:
After self-releasing keys, the key combo states were not being updated if all keys of the combo had been released:

https://github.com/RobertWHurst/Keystrokes/pull/9/files#diff-25a9c55870919ad4e22e73083b9a51f5526667d686815e87aae145f6ba665751R399

At this stage, all keys may be released so `updateState` should indeed proceed with its logic which will result in `isPressed = false`

Reply:
Good catch, this one would have caused other issues too, I'm sure.